pkg/runctl/controller.go

```diff
@@ -177,7 +177,7 @@ func (c *Controller) changeState(pipelineRun k8s.PipelineRun, state api.State) e
 	start := time.Now()
 	oldState, err := pipelineRun.UpdateState(state)
 	if err != nil {
-		klog.Errorf("Failed to UpdateState of [%s] to %q: %q", pipelineRun.String(), state, err.Error())
+		klog.Infof("Failed to UpdateState of [%s] to %q: %q", pipelineRun.String(), state, err.Error())
```

Comment:
Level should be lower, as caller will handle the error.

Reply:
how about set it to 1, like a warning?